icu.yaml

```diff
@@ -1,7 +1,7 @@
 package:
   name: icu
-  version: 74.2
-  epoch: 1
+  version: "75.1" # remove LICENSE file from wolfi source folder when updating as upstream should have fixed the broken symlink
+  epoch: 0
```

Comment:
the upgrade text here is confusing, given LICENSE stuff was already fixed in https://github.com/wolfi-dev/os/pull/18362/files